packages/amplify-category-api/.npmignore

```diff
@@ -1,6 +1,7 @@
 **/__mocks__/**
 **/__tests__/**
 ./src
+!resources/overrides-resource/tsconfig.json
```

Comment:
is this not under cloudformation folder?

Reply:
it is indeed, good catch!